### PR TITLE
Added, updated actions for brand protection.

### PR DESCRIPTION
**Pull request by fr-rpathade**
#### Descriptions:
- Added the following actions:
  - Get Code Repos
  - Get Code Repos Statistics
  - Get Code Repos Matched Domains Statistics
  - Get Rogue Apps
  - Get Rogue App Details by ID
  - Get Takedowns

#### Fix:
- Deprecated following old actions:
  - Get Takedowns
  - Get Rogue Apps
  - Get Rogue App Details by ID

### CODE REVIEW

File: fortinet-fortirecon-brand-protection/info.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
   "name": "fortinet-fortirecon-brand-protection",
   "label": "Fortinet FortiRecon Brand Protection",
-  "version": "2.0.0",
+  "version": "2.1.0",
   "description": "FortiRecon is a Digital Risk Protection Service (DRPS) product that provides an outside-the-network view to the risks posed to your enterprise.",
   "publisher": "Fortinet",
+  "contributor": "Ramkrushna Pathade",
   "cs_approved": true,
   "cs_compatible": true,
   "category": "Attack Surface Management",
```
Comment (cs-omkar-kumbhar) [inline fortinet-fortirecon-brand-protection/info.json:10]: please update the "help_online" key to empty string.
"help_online": ""

File: fortinet-fortirecon-brand-protection/release_notes.md [hunk 2]
```diff
@@ -1,25 +1,15 @@
-#### What's Fixed in 2.0.0
+#### What's Fixed in 2.1.0
```
Comment (cs-omkar-kumbhar) [inline fortinet-fortirecon-brand-protection/release_notes.md:1]: please update the release notes to:
#### What's Fixed in 2.1.0
- Added the following actions:
  - Get Code Repositories
  - Get Code Repos Statistics
  - Get Code Repos Matched Domains Statistics
- In 'Get Rogue Apps' action updated the options for status and added the following parameters: App Store Name, Start Date, End Date.
- In 'Get Takedown Requests' action updated the options for status and renamed the 'Keyword' parameter to 'Search By Keyword'.
- Updated the output schema for following actions: Get Rogue Apps, Get Rogue App By ID, Get Takedown Requests.
- Deprecated the following actions:
  - Get Code Repo Exposures Statistics
  - Get Matched Domains Statistics

**Review by cs-omkar-kumbhar — CHANGES_REQUESTED** [review]

File: fortinet-fortirecon-brand-protection/release_notes.md (hunk 2 above)
Comment (fr-rpathade) [reply fortinet-fortirecon-brand-protection/release_notes.md:1]: Done

File: fortinet-fortirecon-brand-protection/info.json (hunk 1 above)
Comment (fr-rpathade) [reply fortinet-fortirecon-brand-protection/info.json:10]: Done

**Pull request merged**